stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -519,7 +524,10 @@ public fun UploadingFooter(
 ) {
     val attachments = message.attachments
 
-    val uploadedCount = attachments.count { it.uploadState == Attachment.UploadState.Success }
+    val uploadStates: List<StateFlow<Boolean>> = attachments
+        .mapNotNull { it.uploadId }
+        .map { uploadId -> ProgressTrackerFactory.getOrCreate(uploadId).isComplete() }
+    val uploadedCount: Int by combine(uploadStates) { values -> values.count { it } }.collectAsState(initial = 0)
```

Comment:
I'd maybe just rename the `it` to `isUploaded` for clarity, the rest looks good!

Reply:
I wanted to take this suggestion but `values.count { isUploaded -> isUploaded }` is just so ehh